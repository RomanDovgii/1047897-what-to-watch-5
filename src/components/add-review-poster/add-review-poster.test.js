import React from "react";
import renderer from "react-test-renderer";
import AddReviewPoster from "./add-review-poster";

const testPoster = `none`;
const testName = `none`;

it(
    `Should render correct AddReviewPoster`,
    () => {
      const tree = renderer
      .create(
          <AddReviewPoster
            poster={testPoster}
            name={testName}
          />
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);
