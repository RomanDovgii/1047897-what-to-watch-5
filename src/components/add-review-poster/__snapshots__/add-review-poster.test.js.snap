// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct AddReviewPoster 1`] = `
<div
  className="movie-card__poster movie-card__poster--small"
>
  <img
    alt="none"
    height="327"
    src="none"
    width="218"
  />
</div>
`;
